### ISSUE TITLE
Option to support fixed angles

### ISSUE THREAD

**Issue by aguedob** [iVis-at-Bilkent/cytoscape.js-edge-bend-editing#17]
It would be awesome to have an option to support fixed angles in connectors while dragging a node
![dragelaine](https://user-images.githubusercontent.com/11948304/42367387-59ce03d8-8104-11e8-9cb6-9c4979fd21cb.png)

**Comment by chinicuin** [comment]
Any update on this feature? It would be great to have this option. 😃

**Comment by ugurdogrusoz** [comment]
We do not currently have the resources to work on this. Feel free to contribute and do a PR on the unstable branch though. It would be highly appreciated.

**Comment by chinicuin** [comment]
Sure, I will give it a try in the next couple of weeks and do the corresponding pull request.

**Comment by chinicuin** [comment]
https://js.cytoscape.org/#style/taxi-edges
There is an edge type called taxi, I hope this helps @aguedob

**Comment by leolumicrosoft** [comment]
I actually needs this feature in my project. I will try developing it and merge in. Let me know @ugurdogrusoz if it is already implemented in some way.
The feature I plan to achieve is, when mouse dragging a bend anchor point to a location, which is either 0 or 45 or 90 to a anchor point(or source/dest node) before or after the dragging anchor point, it will automatically "stick" to that location.

**Comment by ugurdogrusoz** [comment]
No, it hasn't been implemented and I don't believe it's straightforward.

**Comment by leolumicrosoft** [comment]
I have completely it, I will commit in a short while.  I have tested in the demo html.
As I do not have a comprehensive understanding of the other code, it might need some additional code to avoid confliction. Please review.

**Comment by ugurdogrusoz** [comment]
PR must be to the unstable branch and the new feature needs to be off by default with an option.

**Comment by leolumicrosoft** [comment]
Understood. It is strange that the current unstable branch does not work in my laptop after running webpack again (without any change). so I develop and test this feature in master branch. I will PR to unstable still.
![image](https://user-images.githubusercontent.com/68049677/118464452-73444080-b733-11eb-8a15-dc6fc5c639cf.png)

**Comment by leolumicrosoft** [comment]
oops, sorry I misunderstood the original feature request. ..... I am so sorry, i actually implement a feature that will "stick" anchor to some perfect angle when dragging anchor
it is different from the original feature request, which asking for more node and keep anchor angle..

Shall I start a new issue and paste this implementation. This feature is what I need in my other project.
